### PR TITLE
Bugfix/DE452288 Account Manager Util

### PR DESCRIPTION
**Pull request by abhisheknaidu-462**
In the file AccountManagerUtil.java null pointer exception for **accountType** has been handled.

### CODE REVIEW

File: mas-foundation/src/main/java/com/ca/mas/core/storage/sharedstorage/AccountManagerUtil.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public AccountManagerUtil(Context context, String accountName, boolean sharedSto
             //Attempt to retrieve the account
             Account[] accounts = mAccountManager.getAccountsByType(accountType);
             for (Account account : accounts) {
-                if (accountName.equals(account.name)) {
+                if (accountName != null && accountName.equals(account.name)) {
                     String password = mAccountManager.getPassword(account);
                     String savedPassword = identifier.toString();
                     if (password.equals(savedPassword)) {
```
Comment (nagarajudeshettybroadcom) [inline mas-foundation/src/main/java/com/ca/mas/core/storage/sharedstorage/AccountManagerUtil.java:58]: Add null check to password also as per logs here
https://github.com/CAAPIM/Android-MAS-SDK/issues/340

File: mas-connecta/src/main/java/com/ca/mas/connecta/client/MASConnectaManager.java [hunk 2]
```diff
@@ -68,6 +68,8 @@ private MqttConnecta getMqttConnecta() {
         if ((connectOptions == null) || (connectOptions.getServerURIs() == null)) {
             return new GatewayMqttConnecta();
         } else {
+            if (clientId == null)
+                throw new NullPointerException("clientId");
```
Comment (nagarajudeshettybroadcom) [inline mas-connecta/src/main/java/com/ca/mas/connecta/client/MASConnectaManager.java:72]: I need null check here because not relevant to ticket logs

Comment (abhisheknaidu-462) [reply mas-connecta/src/main/java/com/ca/mas/connecta/client/MASConnectaManager.java]: Made the required changes as per the comments.

**Comment by nagarajudeshettybroadcom** [comment]
@abhisheknaidu-462  can you take a branch from develop and create PR to develop only please stable branch is no longer use

**Review by ppbs2 — APPROVED** [review]

**Pull request closed without merging**